src/rgw/rgw_sal_motr.cc

```diff
@@ -2108,12 +2106,28 @@ int MotrObject::remove_mobj_and_index_entry(const DoutPrefixProvider* dpp, rgw_b
     }
   }
   rc = this->store->do_idx_op_by_name(bucket_index_iname,
-                                              M0_IC_DEL, delete_key, bl);
+                                      M0_IC_DEL, delete_key, bl);
   if (rc < 0) {
     ldpp_dout(dpp, 0) << "Failed to delete object's entry " << delete_key 
                                       << " from bucket index. " << dendl;
     return rc;
   }
+
+  // Subtract object size & count from the bucket stats.
+  if (ent.is_delete_marker())
+    return rc;
+  rc = update_bucket_stats(dpp, this->store, ent.meta.owner, bucket_name,
+                            ent.meta.size, ent.meta.size, 1, false);
+  if (rc != 0) {
+    ldpp_dout(dpp, 20) << __func__ << ": Failed stats substraction for the "
```

Comment:
Change it to `Stats subtraction failed for `

Reply:
sure will address with the next PR regarding Stats.